docs/asyncapi.yaml

```diff
@@ -1,7 +1,7 @@
 asyncapi: 2.4.0
 info:
   title: FFC Payment Statement Generator
-  version: '1.0.0'
+  version: '1.0.1'
   description: Generate PDF payment statements
 channels:
   payment/request:
```

Comment:
As per our AsyncAPI standard doc, we want to use the Topic/Subscription as the name of the channel for this: https://eaflood.atlassian.net/wiki/spaces/SFI/pages/4177723458/AsyncAPI+Structure#Channels